### PR TITLE
Fix/go direct to job page

### PR DESCRIPTION
**Pull request by saraHumphries**
Page now gets the job.id from the url and searches through jobs to get this

### CODE REVIEW

File: src/components/FullJob/index.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ const FullJob = ({job, currentUser, isAdmin, approveJob, unPublishJob}) => {
                 {(job.approved === false && job.creator_id === currentUser.uid) || isAdmin === true?
                     <Link
                         to={{
-                            pathname: ROUTES.EDIT_JOB
+                            pathname: `/my/jobs/${job.id}/edit`
```
Comment (KimberleyCook) [inline src/components/FullJob/index.js:36]: I'm wondering should the single job page URL and this URL read `/my/job/${job.id}/edit` instead?

jobs -> job as it's a single page, also give the main URL will read jobs/codebar.io

File: src/constants/routes.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ export const JOB = '/jobs/:job_id';
 export const MY_JOBS = '/my/jobs';
 export const SUBMIT_JOB = '/my/jobs/new'
 export const PREVIEW_JOB = '/my/jobs/new/preview'
-export const EDIT_JOB = '/my/jobs/job_id/edit'
+export const EDIT_JOB = '/my/jobs/:job_id/edit'
 export const ADMIN_LIST_JOBS = '/admin/jobs'
 export const ADMIN_PREVIEW_DRAFT_JOB = '/admin/jobs/:job_id'
```
Comment (KimberleyCook) [inline src/constants/routes.js:8]: Yeah so my above comment raltes to these. The URLs should read :

`/my/job/new/preview`
`/my/job/:job_id/edit`
`/job/:job_id`

Does that make sense?

Comment (saraHumphries) [reply src/constants/routes.js:8]: I think the standard is to use the plural.  See number 3. here https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/ But can change if you think that makes more sense for this site

Comment (KimberleyCook) [reply src/constants/routes.js:8]: Ah ok, no worries then. We can keep it as it is

**Review by KimberleyCook — APPROVED** [review]

**Pull request merged**